src/main/java/com/schedule/workout/workoutScheduler/controller/workout/WorkoutModel.java

```diff
@@ -0,0 +1,101 @@
+package com.schedule.workout.workoutScheduler.controller.workout;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.schedule.workout.workoutScheduler.database.model.UserDB;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.*;
+
+public class WorkoutModel {
+    private String id;
+    @NotEmpty
+    @Size(min = 2, max = 20)
+    private String name;
+    @NotEmpty
+    @Size(min = 2, max = 80)
+    private String description;
+    @NotNull
+    private Integer duration;
+    @NotNull
+    @Min(value = 1)
+    @Max(value = 5)
+    private Integer difficulty;
+
+    @ManyToOne
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JoinColumn(name = "user_id")
+    private UserDB user;
```

Comment:
For the  model that is used for GET operations, you should not return entire user object, but only necessary parts that should be exposed. Lets say that we want to only expose userId and field name that should hold value of first and last name